### DISCUSSION TITLE
Redundant menu after update

### DISCUSSION THREAD

**Discussion by pappastech** [modoboa/modoboa#2109] [Help]
I just updated Modoboa yesterday and now have redundant 'Statistics' menu items available. The links are identical. Where is the source file for the menu to remove this?

![image](https://user-images.githubusercontent.com/532780/103306487-ab90ef80-49db-11eb-911f-6f235a4eab24.png)

Thanks,
Tom

**Comment by tonioo** [comment]
Make sure you followed upgrade instructions, especially this one: https://modoboa.readthedocs.io/en/latest/upgrade.html#id3.

**Comment by pappastech** [comment]
I followed that as far as I can tell but stats still shows up twice despite not being called out specifically in the settings.py file.

**Comment by tonioo** [comment]
Have you reloaded uwsgi?
